### PR TITLE
Manager donvi

### PR DESCRIPTION
**Pull request by quanlx**
Pls review and merge.

Thanks

### CODE REVIEW

File: Source/EOffice.Entities/Master/User.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {
     public class User
     {
-        public int Id { get; set; }
+        public long Id { get; set; }
```
Comment (phuongvoduc) [inline Source/EOffice.Entities/Master/User.cs]: using `int`

File: Source/EOffice.Entities/Model/Unit.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+using System.ComponentModel.DataAnnotations;
+
+namespace EOffice.Entities.Model
+{
+    public class Unit
+    {
+        public long Id { get; set; }
+
+        [DisplayName("Unit Name")]
+        [Required(ErrorMessage = "Please input this field")]
```
Comment (phuongvoduc) [inline Source/EOffice.Entities/Model/Unit.cs]: pls fix globalization https://github.com/phuongvoduc/EOffice-Web/issues/11

**Comment by phuongvoduc** [comment]
@quanlx fix conflict

**Comment by quanlx** [comment]
@phuongvoduc done

**Pull request merged**